layers/+lang/csharp/config.el

```diff
@@ -1,4 +1,4 @@
-;;; packages.el --- C# Layer configuration File for Spacemacs
+;;; packages.el --- csharp Layer configuration File for Spacemacs
```

Comment:
Please be consistent with the name either use C# everywhere or csharp

Reply:
I changed it on all files to `C#`